src/index.html

```diff
@@ -2,12 +2,305 @@
+              </p>
+            </div>
+
+            <div class="testimonials__person">
+              <img class="testimonials__img" src="./images/people/people__2.svg"
+                alt="Scarlett Hawkins">
+              <p class="testimonials__name">
+                Scarlett Hawkins
+              </p>
+              <p class="testimonials__job">
+                Senior Vice President
+              </p>
+              <p class="testimonials__review">
+                My soul is illuminated by an unearthly joy, as these beautiful
+                spring morning, which I enjoy with all my heart
+              </p>
+            </div>
+
+            <div class="testimonials__person">
+              <img class="testimonials__img" src="./images/people/people__3.svg"
+                alt="Max Henry">
+              <p class="testimonials__name">
+                Max Henry
+              </p>
+              <p class="testimonials__job">
+                Chief Design Officer
+              </p>
+              <p class="testimonials__review">
+                When my lovely valley of steam rises and half-day sun is above
+                an impermeable
+              </p>
+            </div>
+
+          </div>
+
+        </div>
+      </div>
+
+      <div class="contact-us">
+
+        <div class="contact-us__wrapper">
+
+          <h2 class="subtitle">
+            Contact us
+            <div class="subtitle__logo"></div>
+          </h2>
+
+          <div class="contact-us__info">
+
+            <div class="contact-us__container">
+              <div>
+                <div class="contact-us__type">Phone</div>
+                <div class="contact-us__content contact-us__content--1">+1 234 5555-55-55</div>
+                <div class="contact-us__type">Email</div>
+                <div class="contact-us__content">hello@miami.com</div>
+              </div>
+              <div>
+                <div class="contact-us__type">Address</div>
+                <div class="contact-us__content">
+                  400 first ave. n.
+                  <br>
+                  suite 700
+                  <br>
+                  Minneapolis, MN
+                  <br>
+                  55401
+                </div>
+              </div>
+            </div>
+
+            <form class="contact-us__form" action="">
+              <input
+                class="contact-us__input"
+                type="text"
+                placeholder="Name"
+                required
+              >
+              <input
+                class="contact-us__input"
+                type="email"
+                placeholder="Email"
+                required
+              >
+              <textarea
+                class="contact-us__text-area"
+                placeholder="Message"
+                rows="4"
+                cols="1"
+                required
+              >
+              </textarea>
```

Comment:
```suggestion
             ></textarea>
```
так будет отображаться плейхолдер и не будет ругаться линтер